stdlib/Distributed/src/managers.jl

```diff
@@ -178,6 +179,14 @@ function launch_on_machine(manager::SSHManager, machine, cnt, params, launched,
     end
     sshflags = `$(params[:sshflags]) $portopt`
 
+    # First it checks if ssh multiplexing has been already enabled and the master process is running.
+    # If it's already running, later ssh sessions also use the same ssh multiplexing session.
+    # If not, it creates a new ssh multiplexing session to open an ssh tunnel by default.
+    if tunnel && !success(`ssh $sshflags -O check $host`)
+        controlpath = "~/.ssh/julia-%r@%h:%p"
```

Comment:
ControlPath format needs to be documented. The ssh configuration required for this to work should be documented as well.

Reply:
I think multiplexed tunneling should be mentioned in this doc section: https://github.com/JuliaLang/julia/blob/master/doc/src/manual/parallel-computing.md#network-requirements-for-localmanager-and-sshmanager.

Reply:
I added doc about it in ce7135778e7806595eb3e6a04bce17ba40181755